pkg/kubefed2/enable/util.go

```diff
@@ -141,6 +119,37 @@ func LookupAPIResource(config *rest.Config, key, targetVersion string) (*metav1.
 	return nil, errors.Errorf("Unable to find api resource named %q.", key)
 }
 
+func NameMatchesResource(name string, apiResource metav1.APIResource, group string) bool {
+	lowerCaseName := strings.ToLower(name)
```

Comment:
(No action required) Given that the key was previously lowercased outside of the loop, is there a reason not to continue that behavior?

Reply:
In that case the caller of the utility function would need to take the responsibility of passing the lower cased name.